python/taichi/lang/simt/warp.py

```diff
@@ -102,9 +102,11 @@ def match_any():
     pass
 
 
-def match_all():
-    # TODO
-    pass
+def match_all(mask, val):
+    return expr.Expr(
```

Comment:
Can you please replace this based on the proposal here https://github.com/taichi-dev/taichi/pull/4957. Thanks!

Reply:
Sure